pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -628,15 +628,18 @@ def _render_jira_template(fields: Dict, template_vars: Dict):
         return fields
 
     async def set_advisory_dependencies(self, advisories):
-        blocking_kind = ['image', 'extras']
-        target_kind = ['rpm', 'metadata']
-        expected_blocking = {i for i in [advisories[k] for k in blocking_kind if k in advisories] if i}
-        target_advisories = {i for i in [advisories[k] for k in target_kind if k in advisories] if i}
-        if not target_advisories or not expected_blocking:
-            return
-
-        _LOGGER.info(f"Setting blocking advisories ({expected_blocking}) for {target_advisories}")
-        for target_advisory_id in target_advisories:
+        # dict keys should ship after values.
+        blocking_kinds = {
+            'rpm': {'image', 'extras'},
+            'metadata': {'image', 'extras'},
+            'microshift': {'rpm', 'image'},
+        }
+        for target_kind in blocking_kinds:
```

Comment:
```suggestion
        for target_kind in blocking_kinds.keys():
```
Would make this explicit